tests/python/unittest/test_numpy_op.py

```diff
@@ -3131,16 +3134,17 @@ def hybrid_forward(self, F, a, b, *args, **kwargs):
         'mod': (1.0, 5.0, None, None),
         'power': (1.0, 3.0, lambda y, x1, x2: _np.power(x1, x2 - 1.0) * x2,
                             lambda y, x1, x2: _np.power(x1, x2) * _np.log(x1)),
-        'equal': (0.0, 2.0, None, None),
-        'not_equal': (0.0, 2.0, None, None),
-        'greater': (0.0, 2.0, None, None),
-        'less': (0.0, 2.0, None, None),
-        'greater_equal': (0.0, 2.0, None, None),
-        'less_equal': (0.0, 2.0, None, None),
-        'logical_and': (0.0, 2.0, None, None),
-        'logical_or': (0.0, 2.0, None, None),
-        'logical_xor': (0.0, 2.0, None, None),
     }
+    if not has_tvm_ops():
+        funcs['equal'] = (0.0, 2.0, None, None)
+        funcs['not_equal'] = (0.0, 2.0, None, None)
+        funcs['greater'] = (0.0, 2.0, None, None)
+        funcs['less'] = (0.0, 2.0, None, None)
+        funcs['greater_equal'] = (0.0, 2.0, None, None)
+        funcs['less_equal'] = (0.0, 2.0, None, None)
+        funcs['logical_and'] = (0.0, 2.0, None, None)
+        funcs['logical_or'] = (0.0, 2.0, None, None)
+        funcs['logical_xor'] = (0.0, 2.0, None, None)
```

Comment:
I think there's a route to tvmop when it is enabled, can we disable such route for mixed-type operation? changing test does not seem to be a correct fix.